### DISCUSSION TITLE
question: Are we using Discussions?

### DISCUSSION THREAD

**Discussion by rjc3rd** [Ahwxorg/LibreY#160] [General]
Are we using Discussions? I want to discuss  aspects of LibreY that I don't feel are "Issues" 

It seems the only discussions are over on the Issues section of the github.

**Comment by davidovski** [comment]
Go for it. We also have a matrix room at #librey:ahwx.org if you want to keep it more informal

**Reply by rjc3rd** [reply]
I guess I'd like to keep it more formal. I don't use matrix, so I'd prefer to use this discussion area as long as it's alright with everyone else. And really only if others will even participate here. I think they would though, since everyone here is already on github.

**Reply by Ahwxorg** [reply]
Sure! I get emails for that and check GitHub quite often so that's fine by me :) (though the Matrix room is advised to join in case of a security vulnerability.)

**Comment by rjc3rd** [comment]
I guess I'll close this discussion now then.
